packages/bot-web-ui/src/components/journal/journal.tsx

```diff
@@ -236,18 +265,18 @@ const Journal = ({
         >
             <Tools {...props} />
             <div className='journal__item-list'>
-                {filtered_messages.length ? (
+                {filtered_messages_length ? (
                     <DataList
                         className='journal'
                         data_source={filtered_messages}
-                        rowRenderer={args => <JournalItem {...args} />}
-                        keyMapper={row => row.unique_id}
+                        rowRenderer={(args: TJournalDataListArgs) => <JournalItem {...args} />}
+                        keyMapper={(row: TFilterMessageValues) => row.unique_id}
                     />
                 ) : (
                     <>
                         {contract_stage >= contract_stages.STARTING &&
-                        !!props.checked_filters.length &&
-                        !unfiltered_messages.length &&
+                        !!Object.keys(props.checked_filters as TCheckedFilters).length &&
```

Comment:
can we destructure checked_filters as well? like we did all other parameters from the props? @maryiafrantsava-binary

Reply:
ok, I will add destructuration, thank you 🙏